spec/models/mixins/supports_feature_mixin_spec.rb

```diff
@@ -262,18 +254,72 @@ class MegaPost
   end
 
   context "feature that is implicitly unsupported" do
-    it "class responds to supports_feature?" do
-      expect(Post.supports_nuke?).to be false
+    it "is unsupported by the class" do
+      expect(Post.supports_delete?).to be false
```

Comment:
`delete` is not implicitly unsupported, but explicitly on the module `Post::Operations::Publishing`
thats why this specs pass